requirements.txt

```diff
@@ -1,20 +1,20 @@
 #
-# This file is autogenerated by pip-compile with python 3.11
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
The version can matter ( I don't think it will in this case) since it's on this version the tool will base the requirements